src/pages/components/tile/usage.mdx

```diff
@@ -487,91 +528,34 @@ version.
-
-<Caption>An example of the standard layout</Caption>
-
-</Column>
-</Row>
-
-<Row>
-<Column colLg={4}>
-
-![Vertical masonry layout for tiles](/images/tile-usage-18.png)
-
-<Caption>Vertical masonry layout for tiles</Caption>
-
-</Column>
-
-<Column colLg={4}>
+<Column colLg={8}>
 
-![An example of the vertical masonry layout](images/tile-usage-19.png)
+![Layouts for tiles](/images/tile-usage-18.png)
 
-<Caption>An example of the vertical masonry layout </Caption>
+<Caption>
+  Examples of tile layouts: standard, vertical masonry, and horizontal masonry
+</Caption>
 
 </Column>
 </Row>
 
-<Row>
-<Column colLg={4}>
-
-![Horizonal masonry layout for tiles](/images/tile-usage-20.png)
-
-<Caption>Horizonal masonry layout for tiles</Caption>
-
-</Column>
-
-<Column colLg={4}>
+## Universal behaviors
 
-![An example of the horizonal masonry layout](images/tile-usage-21.png)
+The behaviors listed in this section are universal across all variants. For
+behaviors that are unique to each variant, see each of the component variants
+sections below.
 
-<Caption>An example of the horizonal masonry layout </Caption>
-
-</Column>
-</Row>
-
-### Elevation
-
-Tiles reside in the same plane as the background layer—they do not have
-elevation. Do not add drop shadows to tiles. Tiles organize essential
-information and have the same visual hierarchy as content within the same page.
-Modals, popovers, and dialogs do have elevation but are not inherently part of
-the user's main workstream and are invoked on screen to reveal secondary
-information, actions, or notifications.
+### States
 
-### Call-to-actions
+For detailed visual information about the various states for this component, see
+the [Style](http://localhost:8000/components/tile/style) tab.
 
-For **base, selectable, and expandable tiles**, use `$link-01` when icon and
-link are paired together or use `$interactive-04` when the icon stands alone.
-
-For **clickable tiles**, don’t highlight text or icons to avoid them being
-confused as links or clickable.
-
-<DoDontRow>
-
-  <DoDont type="do" caption="Do use text or icon for clickable tiles if needed.">
-
-![Do use text or icon for clickable tiles if needed.](images/tile-usage-22.png)
-
-  </DoDont>
-  <DoDont type="dont" caption="Do not use link or highlight the icon color for clickable tiles.">
-
-![Do not use link or highlight the icon color for clickable tiles.](images/tile-usage-23.png)
-
-  </DoDont>
-</DoDontRow>
-
-## Behaviors
+| State      | When to use                                                                                                                                         |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Enabled_  | When a tile is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component    |
```

Comment:
The image under this table should say `Multi-select tile` for the last tile.

<img width="702" alt="Screen Shot 2023-02-14 at 9 18 15 AM" src="https://user-images.githubusercontent.com/43969356/218780119-87d284d2-1d2b-4efd-a1f2-13130c2a99f5.png">